spec/index.bs

```diff
@@ -13222,7 +13221,7 @@ which allows changing the canvas configuration without replacing the canvas.
 ## GPUCanvasContext ## {#canvas-context}
 
 <script type=idl>
-[Exposed=(Window, DedicatedWorker, ServiceWorker), SecureContext]
```

Comment:
Depending on the answer to the comment below, maybe GPUCanvasContext should only be exposed to DedicatedWorker and not ServiceWorker/SharedWorker.

Reply:
OffscreenCanvas should almost certainly be supported (if not transferControlToOffscreen) so marking this as resolved.